app/code/Magento/PageCache/Observer/SwitchPageCacheOnMaintenance/PageCacheState.php

```diff
@@ -1,74 +0,0 @@
-<?php
```

Comment:
Please revert changes in this file as this is not observer and removing it will be backward incompatible. If it’s not used anymore - please mark it with tag `@deprecated` with description w